pkgs/by-name/ma/maltego/package.nix

```
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, giflib
+, gawk
+, makeBinaryWrapper
+, icoutils
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation rec {
```

We try to avoid the `rec`, and in the case of `mkDerivation` you can do something like this instead:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```
don't forget the closing `)`.
You can then use finalAttrs.var which is more explicit and less dependent on context.

---

Fixed